README.md

```diff
@@ -1 +1,826 @@
+        
+  >This guide provides instructions on how Ballerina can be used to integrate with Stream-Processor Message Tracer.
+  
+The following are the sections available in this guide.
+
+- [What you'll build](#what-you’ll-build)
+- [Prerequisites](#prerequisites)
+- [Implementation](#implementation)
+- [Testing](#testing)
+- [Testing with distributed message tracing](#testing-with-distributed-message-tracer.)
+     - [Traces](#views-of-traces)
+
+## What you’ll build
+To perform this integration with Distributed Message Tracer, a real world use case of a very simple student management system is used.
+This system will illustrate the manipulation of student details in a school/college management system. The administrator will be able to perform the following actions in this service.
+
+    - Add a student's details to the system.
+    - List down all the student's details who are registered in the system.
+    - Delete a student's details from the system by providing student ID.
+    - Generate a mock error (for observability purposes).
+    - Get a student's marks list by providing student ID.
+
+![Honeycomb](images/ballerina-sp.svg "Ballerina-Honeycomb")
+
+- **Make Requests** : To perform actions on student  management service, a console based client program has been written in Ballerina for your ease of making requests.
+
+## Prerequisites
+ 
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- [MySQL](https://github.com/ballerina-guides/message-tracing-with-stream-processor/blob/master/resources/testdb.sql)
+- [WSO2 - Stream Processor v4.3.0 or above](https://github.com/wso2/product-sp/releases)
+- A Text Editor or an IDE 
+
+## Implementation
+
+> If you want to skip the basics, you can download the GitHub repo and continue from the [Testing](#testing) section.
+
+### Implementing database
+ - Start MySQL server in your local machine.
+ - Create a database with name `testdb` in your MySQL localhost. If you want to skip the database implementation, then directly import the [testdb.sql](https://github.com/ballerina-guides/message-tracing-with-stream-processor/blob/master/resources/testdb.sql) file into your localhost. You can find it in the Github repo.
+
+### Create the project structure
+        
+ For the purpose of this guide, let's use the following package structure.
+ 
+    message-tracing-with-stream-processor
+           └── guide
+                ├── students
+                │   ├── student_management_service.bal
+                │   ├── student_marks_management_service.bal
+                ├── client_service
+                |         └── client_main.bal
+                └── ballerina.conf
+        
+
+- Create the above directories in your local machine, along with the empty `.bal` files.
+
+- You have to add the following lines in your [ballerina.conf](https://github.com/ballerina-guides/message-tracing-with-stream-processor/blob/master/ballerina.conf).
```

Comment:
Let's just say "Add the following lines in..." here.

Also, shall we reformat this to be a numbered list?